gettingstarted/InstallingEquella.md

```diff
@@ -44,24 +51,26 @@ The primary log to watch is located under ```{install_location}/logs/resource-ce
 and/or
 ~$ ./manager start
 ```
-To stop openEQUELLA or the manager, run the same command to start the application, but pass in 'stop' instead of 'start'.
+To stop openEQUELLA, run the same command to start the application, but pass in 'stop' instead of 'start'.
 
-### Windows
-Assuming openEQUELLA and the openEQUELLA Manager have been installed as services, navigate to your services, and select openEQUELLA to start.
+**Note:** The primary log to watch is located under: 
+
+{install_location}/logs/resource-centre/{current_date}/application.{X.}html
```

Comment:
I recommend you enclose this line in back ticks to make it mono-space.

Reply:
👍